models/recruit/src/index.ts

```diff
@@ -80,6 +76,10 @@ export class TApplicant extends TDocWithState implements Applicant {
 
   @Prop(TypeString(), 'Assigned recruiter' as IntlString)
   employee!: Ref<Employee>
+
+  // We need this two to make typescript happy.
+  declare state: TDocWithState['state']
```

Comment:
Probable we could somehow exclude them for typescript with types, but I've not found an easy way.